exec/java-exec/src/main/java/org/apache/drill/exec/cache/VectorAccessibleSerializable.java

```diff
@@ -57,6 +57,12 @@
   private BatchSchema.SelectionVectorMode svMode = BatchSchema.SelectionVectorMode.NONE;
   private SelectionVector2 sv2;
 
+  /**
+   * Disk I/O buffer used for all reads and writes of DrillBufs.
+   */
+
+  private byte buffer[] = new byte[32*1024];
```

Comment:
(1) Is 32K too small ? (because it is heap?) Does a higher size perform better ?
(2) Would it work better if the operator would allocate this serialization buffer from the direct memory (and only when spilling), and then free it at the end ?